### ISSUE TITLE
Installation instructions

### ISSUE THREAD

**Issue by shumafuk** [zesterer/openmw-volumetric-clouds#2]
I'm little bit confused about install instructions.

- Isn't distance fog disabled by default? If not, should my settings.cfg in documents/my games/openmw have this lines?
`[Fog Settings]
use distant fog = false`
- How can I enable FXAA and change render order? I probably don't know how can I configure shaders in OpenMW at all.
- I have multiple data entries in my openmw.cfg in documents/my games/openmw, should I enter a new one with path to openmw-volumetric-clouds?
- How can I enable post-process shaders in settings.cfg? Should it have those lines?
`[Post Processing Settings]
enabled = true`

After press F2 in game, nothing happens. Any advice is appreciated. Thanks.

**Comment by zesterer** [comment]
> After press F2 in game, nothing happens. Any advice is appreciated.

This sounds a lot like you don't have the latest *nightly* build of OpenMW. The post-processing branch was only merged a few days ago, so only nightly builds from the last few days can be used.

**Comment by shumafuk** [comment]
You are right, I did not have latest nightly version. After update, it works. Thanks!
